Examples/KeyboardSupport-iOSExample/AutoNavigatorViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  AutoNavigatorViewController.swift
+//  KeyboardSupport
+//
+//  Created by John Davis on 12/18/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import UIKit
+import KeyboardSupport
+
+/// The `AutoNavigatorViewController` demonstates the configuration and use of a  `KeyboardAutoNavigator` instance.
+///
+/// Note that the `KeyboardAutoNavigator` will disable and enable the next back buttons in a `NavigatingKeyboardAccessory` when no previous or next input field is available fornavigation.
```

Comment:
Minor typos in the comment: "enable the next back buttons" --> "enable the next/back buttons" and "fornavigation" --> "for navigation".

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/170fbc27225cb347e97c16d4b37ada8486816355